molecule/driver/delegated.py

```diff
@@ -150,21 +159,19 @@ def ansible_connection_options(self, instance_name):
         if self.managed:
             try:
                 d = self._get_instance_config(instance_name)
+                conn_dict = {}
+                conn_dict['ansible_user'] = d.get('user')
+                conn_dict['ansible_host'] = d.get('address')
+                conn_dict['ansible_port'] = d.get('port')
+                conn_dict['ansible_connection'] = d.get('connection', 'smart')
```

Comment:
Will this default trip anyone up? Not sure what it's doing. Document it?

Reply:
```
ansible_connection
Connection type to the host. This can be the name of any of ansible’s connection plugins. SSH protocol types are smart, ssh or paramiko. The default is smart. Non-SSH based types are described in the next section.
```
Already documented by ansible team: https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html

Reply:
Nice one, thanks.